index.html

```diff
@@ -8,8 +8,12 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="shortcut icon" href="images/lightbeam-48.png" type="image/x-icon">
     <link rel="icon" href="images/lightbeam-48.png" type="image/x-icon">
+    <link rel="stylesheet" href="css/style.css" type="text/css">
+    <script src="js/store.js" type="text/javascript"></script>
+    <script src="js/viz.js" type="text/javascript"></script>

```

Comment:
Not sure if this is the right way to add script tags, inspite of adding these two files in `manifest`.
If I do not add them, `viz` object cannot get the reference to `getElementById` from this page to draw the graph.